### ISSUE TITLE
[1.12.2 2847 with ForgeGradle 2.3] Build fails with native Exception on macOS

### ISSUE THREAD

**Issue by anatawa12** [OpenCubicChunks/OptiFineDevTweaker#5]
Crashed with this message

```
2021-04-24 22:26:43.166 java[92666:1337267] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
*** First throw call stack:
(
        0   CoreFoundation                      0x00007fff205306af __exceptionPreprocess + 242
        1   libobjc.A.dylib                     0x00007fff202683c9 objc_exception_throw + 48
        2   CoreFoundation                      0x00007fff20558a9a -[NSException raise] + 9
        3   AppKit                              0x00007fff22cff460 -[NSWindow(NSWindow_Theme) _postWindowNeedsToResetDragMarginsUnlessPostingDisabled] + 321
        4   AppKit                              0x00007fff22d277db -[NSThemeFrame setStyleMask:] + 142
        5   AppKit                              0x00007fff22d27571 __25-[NSWindow setStyleMask:]_block_invoke + 2057
        6   AppKit                              0x00007fff22d26d0e NSPerformVisuallyAtomicChange + 132
        7   AppKit                              0x00007fff22d26c1b -[NSWindow setStyleMask:] + 170
        8   liblwjgl.dylib                      0x0000000130d48ee2 Java_org_lwjgl_opengl_MacOSXDisplay_nSetResizable + 82
        9   ???                                 0x0000000108b8a407 0x0 + 4441285639
        10  ???                                 0x0000000108b79ffd 0x0 + 4441219069
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

[debug.log](https://github.com/OpenCubicChunks/OptiFineDevTweaker/files/6369663/debug.log)
[latest.log](https://github.com/OpenCubicChunks/OptiFineDevTweaker/files/6369664/latest.log)
[gradle-log.txt](https://github.com/OpenCubicChunks/OptiFineDevTweaker/files/6369667/gradle-log.txt)

Optifine: OptiFine_1.12.2_HD_U_G5.jar
OptiFineDevTweaker: aa_do_not_rename_OptiFineDevTweaker-2.5.1-all.jar

**Comment by anatawa12** [comment]
At Config.java#L2666, setResizable was called, and in the function, this exception was happened

**Comment by Niko-sk2x** [comment]
Not much I can do about it. This may be fixed in latest forge for 1.12.2 but if it happens in optifine code, it would have to be fixed in optifine. This is a known issue that seems to happen on JVM versions built for newer macOS versions.

Apple is moving away from supporting old stuff, and 1.12.2 is only going to be more and more broken on macOS going forward.

**Comment by anatawa12** [comment]
Sorry, I've found an issue like this on Optifine's issue tracker.
